index.js

```diff
@@ -560,14 +560,18 @@ exports.generate_key = util.deprecate(function (options) {
  */
 exports.generateSecretASCII = function generateSecretASCII (length, symbols) {
   var bytes = crypto.randomBytes(length || 32);
-  var set = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXTZabcdefghiklmnopqrstuvwxyz';
+  return encodeASCII(bytes, symbols);
+};
+
+function encodeASCII (bytes, symbols) {
+  var set = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXTZabcdefghijklmnopqrstuvwxyz';
```

Comment:
`T` appears twice. This is almost certainly a mistake (that existed before your PR). Replace second `T` with `Y`.

Reply:
I see you added the `j`. :shipit: 💯

Reply:
Good catch, I replaced the second T with a Y!